core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/epochconversion/EpochToLocalDateUnitTest.java

```diff
@@ -0,0 +1,38 @@
+package com.baeldung.epochconversion;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EpochToLocalDateUnitTest {
+
+    @Test
+    void givenEpoch_thenComputeLocalDateCorrectly() {
+        ZoneId zoneId = ZoneId.systemDefault();
+        LocalDate actualDate = LocalDate.of(1995, 4, 15);
+        long date = actualDate.atStartOfDay(zoneId).toInstant().toEpochMilli();
+        LocalDate computedDate =
+                Instant.ofEpochMilli(date)
+                        .atZone(zoneId)
+                        .toLocalDate();
+        assertEquals(actualDate, computedDate);
```

Comment:
In assertEquals, the first argument should be the expected, and the second argument should be the actual.
Also let's add a blank line between the given/when and when/then parts of each test where applicable.

Reply:
I have renamed the variables according to the convention and also put an extra line where applicable.